bin/cache_daemon.ml

```diff
@@ -25,8 +25,13 @@ let doc = "Manage the shared artifacts cache"
 
 let info = Term.info name ~doc ~man
 
-let start ~exit_no_client ~foreground ~port_path ~root =
-  let show_endpoint ep = Printf.eprintf "listening on %s\n%!" ep in
+let start ~exit_no_client ~foreground ~port_path ~root ~quiet =
+  let show_endpoint ep =
+    if quiet then
+      Printf.printf "%s\n%!" ep
+    else
+      Printf.eprintf "listening on %s\n%!" ep
```

Comment:
Why the distinction between stdout and stderr here?

Reply:
If we're going to output something in output mode, why not just output the same thing in both cases btw.

Reply:
Initially everything was on stdout. The point here is to be able to parse the output of `dune cache start` to get the endpoint of the freshly started or previously running daemon, and that belongs on stdout.

Reply:
But aren't you printing to stdout conditionally here? Should the stderr be parsed if `quiet` is false?

Reply:
It shouldn't be parsed if quiet is false. You moved that message to stderr, so I just didn't want to undo that, and add something so we can easily parse the output

Reply:
I see. The flow here is still a bit confusing. Something that would be simpler IMO:

* Have an `--output-address` to explicit ask for this information to be printed. This would replace `--quiet`.

* Have `stderr` logging be controlled by `--display` only.

Reply:
PS: `--display script` is fine too.